Tests/AppTests/Middleware/ClientMiddlewareTests.swift

```diff
@@ -0,0 +1,106 @@
+import XCTest
+import Testing
+@testable import Vapor
+@testable import App
+
+class ClientMiddlewareTests: TestCase {
+  let droplet = try! Droplet.testable()
+
+  let validClientToken = "test"
+  let invalidClientToken = String.invalidRandomToken
+
+  func testMiddleware_MiddlewarePresentInConfig() {
+    let config = droplet.config
+    let configMiddlewares = try! config.resolveMiddleware()
+    XCTAssertTrue(configMiddlewares.contains { middleware in
+      if let _ = middleware as? ClientMiddleware {
+        return true
+      }
+      return false
+    })
+  }
+
+  func testMiddleware_ClientTokenIsTakenFromConfig() {
+    let config = droplet.config
+    XCTAssertEqual(config["server", "client-token"]?.string, validClientToken)
```

Comment:
I'm thinking, do we need this test? 

Because the subscript on config is already tested by vapor for sure. So you always gonna take needed parameter from config. The only mistake could be if you mistyped names, but this will be human error and this test won't help then.

What do you think?

Reply:
Fixed. Removed.